### ISSUE TITLE
Another option of code highlight library 

### ISSUE THREAD

**Issue by humorless** [yogthos/markdown-clj#150]
I found this [library ](https://github.com/venantius/glow)

I have tried this and think that is it possible to use it in `markdown-clj`? Will it make the code highlight even better?

**Comment by yogthos** [comment]
Yeah, I think that could be a nice option. Any chance you'd be up for making a pr for it. I think would be good to toggle the feature using a config key. Something like `:colorize?` maybe?

**Comment by humorless** [comment]
Great. I will take some time to work on this.

**Comment by yogthos** [comment]
👍
